pandas/core/window/expanding.py

```diff
@@ -589,7 +586,7 @@ def quantile(
     )
     def cov(
         self,
-        other: Optional[Union[np.ndarray, FrameOrSeriesUnion]] = None,
+        other: Optional[FrameOrSeriesUnion] = None,
```

Comment:
How is this change correct? Or is the docstring not correct? (which says that other can also be an ndarray)

Reply:
At least on master, passing a numpy array raises

```
In [1]: pd.Series(range(10)).expanding().cov(np.arange(10))
Exception: Input arrays must be of the same type!
```
as well there's a test, `test_different_input_array_raise_exception` that checks that an `Exception` is raised with a numpy array.

So I can follow up fixing the documentation